modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/resource/v1_0/PostalAddressResourceImpl.java

```diff
@@ -61,6 +64,25 @@ public void deletePostalAddress(Long postalAddressId) throws Exception {
 		_addressService.deleteAddress(postalAddressId);
 	}
 
+	@Override
+	public void deletePostalAddressByExternalReferenceCode(
+			String postalAddressExternalReferenceCode)
+		throws Exception {
+
+		Address address =
+			_addressLocalService.getAddressByExternalReferenceCode(
+				postalAddressExternalReferenceCode,
+				contextCompany.getCompanyId());
+
+		if (address == null) {
+			throw new NoSuchAddressException(
+				"No Address found with external reference code " +
```

Comment:
can you just check if in other places we put the name of the model (like `Address` here lowercase or not?